services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx

```diff
@@ -251,10 +251,8 @@ describe("<DashboardPage />", () => {
     });
 
     test("Cannot unarchive a WP", async () => {
-      const archiveButton = screen.getAllByRole("button", {
-        name: "Archived",
-      })[0];
-      expect(archiveButton).toBeDisabled();
+      const archivedText = screen.getAllByText("Archived")[1];
```

Comment:
Using `getAllByText` is awkward since it'll pick up text anywhere. You could use `gridcell` in place of `button` to more closely mimic the previous test.

```
screen.getAllByRole("gridcell", {
  name: "Archived",
})[0];
```

Reply:
but the `gridcell` one isn't the one I'm looking for, that's the `status`. The `archived` text I'm looking for is the one after it in the same row.

Reply:
That one also seems to be in a `gridcell`:

<img width="419" alt="Screenshot 2024-08-23 at 4 07 59 PM" src="https://github.com/user-attachments/assets/dbcc9c2d-feaa-4ebd-88e5-117bf4e6cff1">

Maybe instead we could test that there's always two of these? Or that the last `gridcell` is the one with `Archived`?

```
const cells = screen.getAllByRole("gridcell");
const lastCell = cells[cells.length - 1];
expect(lastCell).toHaveTextContent("Archived");
```

Reply:
Another fancy way to write that example:
```
const lastCell = screen.getAllByRole("gridcell").pop();
expect(lastCell).toHaveTextContent("Archived");
```

Reply:
I made that change to the desktop view, but the mobile view isn't in a table like the desktop view is.